compiler/driver.c

```diff
@@ -115,6 +126,14 @@ static void read_options(struct options * o, int argc, char * argv[]) {
                 continue;
             }
 #endif
+#ifndef DISABLE_Go
```

Comment:
It's not consistent with `DISABLE_GO` elsewhere so you'd have to define both, but I don't think anything actually makes use of all the ability to disable backends from the build.  It doesn't seem all that useful as none of the backends drag in extra dependencies.  Certainly nothing in the repo defines these; I've been meaning to check on the mailing list if anyone is making use of them in a wider context and if not to drop them.